cluster/gce/gci/configure-helper.sh

```diff
@@ -1593,4 +1593,5 @@ else
 fi
 reset-motd
 prepare-mounter-rootfs
+modprobe configs
```

Comment:
Good point. How are folks expected to discover the doc you posted? Can you add an entry for `/proc/configz`?